### PR TITLE
VPN-6115: Remove VPN Speed Tests

### PR DESCRIPTION
**Pull request by oskirby**
## Description
The speed test feature of the VPN has not really delivered the value we had hoped it would. Telemetry reports that very few users have been making use of it, and those that do frequently encounter errors, bugs, or misleading information; and there are many other tools that users can take advantage of to perform a similar test.

Therefore, we have decided that the best course of action is to remove the speed test feature from the VPN client.

## Reference
JIRA ticket: [VPN-6115](https://mozilla-hub.atlassian.net/browse/VPN-6115)
Bugzilla issue: [1875139](https://bugzilla.mozilla.org/show_bug.cgi?id=1875139) 

## Checklist
    
- [ ] My code follows the style guidelines for this project
- [ ] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [ ] I have performed a self review of my own code
- [ ] I have commented my code PARTICULARLY in hard to understand areas
- [ ] I have added thorough tests where needed

[VPN-6115]: https://mozilla-hub.atlassian.net/browse/VPN-6115?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/ui/screens/home/controller/connectionInfo/ConnectionInfoScreen.qml [hunk 1]
```diff
@@ -1,234 +0,0 @@
...
-
-        timer.setTimeout(function() {
-            // Finished opening/closing transition
-            isTransitioning = false;
-        }, transitionDuration);
-    }
-
-    onStateChanged: () => {
-        if (state === "open-loading") {
-            Glean.impression.speedTestLoadingScreen.record({
```
Comment (lesleyjanenorton) [inline src/ui/screens/home/controller/connectionInfo/ConnectionInfoScreen.qml:112]: I _think_ we can remove these pings now. @mcleinman is there a reason to keep these in `metrics.yaml` if we are no longer sending?

**Review by lesleyjanenorton — APPROVED** [review]
Spring cleaning! R+ with a few comments/questions

File: src/ui/screens/home/controller/connectionInfo/ConnectionInfoScreen.qml (hunk 1 above)
Comment (mcleinman) [reply src/ui/screens/home/controller/connectionInfo/ConnectionInfoScreen.qml:112]: No, we can remove the metrics. Pedantic, but: I don't believe we can remove any pings (the sending of a group of metrics), but the metrics themselves can be removed here and in `metrics.yaml`.

Comment (lesleyjanenorton) [reply src/ui/screens/home/controller/connectionInfo/ConnectionInfoScreen.qml:112]: thank you! updated my comment to conform to appropriate nomenclature!

File: src/ui/resources/animations/vpnlogo-kinetic_animation.json [hunk 2]
```diff
@@ -1 +0,0 @@
...4,"cix":2,"bm":0,"ix":1,"mn":"ADBE Vector Group","hd":false}],"ip":-36,"op":452,"st":-36,"bm":0},{"ddd":0,"ind":6,"ty":4,"nm":"Logo 1","sr":1,"ks":{"o":{"a":0,"k":100,"ix":11},"r":{"a":0,"k":0,"ix":10},"p":{"a":0,"k":[300,300,0],"ix":2,"l":2},"a":{"a":0,"k":[0,0,0],"ix":1,"l":2},"s":{"a":0,"k":[100,100,100],"ix":6,"l":2}},"ao":0,"shapes":[{"ty":"gr","it":[{"ty":"rc","d":1,"s":{"a":0,"k":[600,600],"ix":2},"p":{"a":0,"k":[0,0],"ix":3},"r":{"a":0,"k":0,"ix":4},"nm":"Rectangle Path 1","mn":"ADBE Vector Shape - Rect","hd":false},{"ty":"tr","p":{"a":0,"k":[0,0],"ix":2},"a":{"a":0,"k":[0,0],"ix":1},"s":{"a":0,"k":[100,100],"ix":3},"r":{"a":0,"k":0,"ix":6},"o":{"a":0,"k":100,"ix":7},"sk":{"a":0,"k":0,"ix":4},"sa":{"a":0,"k":0,"ix":5},"nm":"Transform"}],"nm":"Logo 1","np":1,"cix":2,"bm":0,"ix":1,"mn":"ADBE Vector Group","hd":false}],"ip":-36,"op":453,"st":-36,"bm":0}]}],"layers":[{"ddd":0,"ind":1,"ty":0,"nm":"Logo 1","refId":"comp_0","sr":1,"ks":{"o":{"a":0,"k":100,"ix":11},"r":{"a":0,"k":0,"ix":10},"p":{"a":0,"k":[426,427,0],"ix":2,"l":2},"a":{"a":0,"k":[300,300,0],"ix":1,"l":2},"s":{"a":0,"k":[100,100,100],"ix":6,"l":2}},"ao":0,"w":600,"h":600,"ip":0,"op":452,"st":0,"bm":0}],"markers":[]}
```
[the remark this answers is not in the document]
Comment (oskirby) [reply src/ui/resources/animations/vpnlogo-kinetic_animation.json:1]: Done!

File: src/ui/screens/home/controller/connectionInfo/ConnectionInfoScreen.qml (hunk 1 above)
Comment (oskirby) [reply src/ui/screens/home/controller/connectionInfo/ConnectionInfoScreen.qml:112]: I have removed the metrics

**Pull request merged**